src/screens/ENSAssignRecordsSheet.js

```diff
@@ -114,14 +114,10 @@ export default function ENSAssignRecordsSheet() {
   const { result: dominantColor } = usePersistentDominantColorFromImage(
     avatarUrl || initialAvatarUrl || params?.externalAvatarUrl || ''
   );
-  const [prevDominantColor, setPrevDominantColor] = useState(dominantColor);
 
   useEffect(() => {
-    setAccentColor(dominantColor || prevDominantColor || colors.purple);
```

Comment:
not 100% sure of the correct use of `prevDominantColor`, I wasn't able to repro any problem without it

@jxom any situation that this was made for?

Reply:
this was to avoid flickering of accent color:

https://user-images.githubusercontent.com/7336481/166393624-f383b587-25da-48f9-87fd-f09bf2cefbb8.mp4

Reply:
ah ok makes sense, will take a look

Reply:
fixed with https://github.com/rainbow-me/rainbow/pull/3287/commits/9753746c91354631a154d2de25bd1d4f30ae50ff with this there's only an update of color if the accentColor is set or not accentColor but also no image to get the accent color from. I was seeing that the flickering happened because there's a moment when we have an image url but the accent color is not ready yet, so we were setting the default color while the correct accent color was processed.